### ISSUE TITLE
No matching results for string in select search variant is not communicated via VoiceOver on iOS

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#1905]
# Bug Report

## Description
When typing in a search value where there are no results, the No matching results for ${string} is not announced to the user when it is displayed.

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-core/#/tests/terra-form-select/form-select/controlled-search
2. Type in random string, "asdf"
3. No matching results for asdf is displayed visually but not communicated via VO.

## Expected Behavior
The result of a search should be communicated to screen reader.

## Possible Solution
Might look into using `aria-live` to [communicate search results on the fly](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)

## Environment

* Component Name and Version: terra-select search variant
* Browser Name and Version: Mobile Safari iOS 12 w/ VoiceOver
